packages/ra-core/src/form/FormDataConsumer.spec.tsx

```diff
@@ -26,6 +34,7 @@ describe('FormDataConsumerView', () => {
     it('calls its children function with scopedFormData and getSource if it received an index prop', () => {
         const children = jest.fn(({ getSource }) => {
             getSource('id');
+            return null;
```

Comment:
FMI, why did you need to change this?

Reply:
It was throwing a TypeScript error on children usage